Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "{{project-name}}"
+name = "cosmwasm-template"
```

Comment:
We need to leave these all as templates. You can run `cargo generate` and the generated file will have the proper values filled out.